packages/web-components/src/components/cbp-tooltip/cbp-tooltip.tsx

```diff
@@ -1,30 +1,32 @@
-import { Component, Prop, Element, Host, h } from '@stencil/core';
-import { setCSSProps } from '../../utils/utils';
+import { Component, Prop, Element, Host, h, Listen } from '@stencil/core';
+import { setCSSProps, createNamespaceKey } from '../../utils/utils';
 
 @Component({
   tag: 'cbp-tooltip',
   styleUrl: 'cbp-tooltip.scss'
 })
 export class CbpTooltip {
 
-  
   @Element() host: HTMLElement;
   /** When set, specifies that the tooltip is open */
   @Prop({ reflect: true }) open: boolean = false;
- 
-  //todo: update from alignment to placement
+
+  /** used to set styles for the definition link for text controls*/
+  @Prop({ reflect: true }) definitionLinkStyle: boolean = false;
```

Comment:
I know naming is hard but this is way too verbose. I try to keep things to a single word to act more like HTML and not have a bunch of hyphens. I think I'd go with `variant: 'default' | 'definition' = 'default'` (then you don't need to use the "default" attribute selector when setting styles, only `[variant=definition]`. This also leaves room for expansion in case we do want to add a true link style (or something else) later.